packages/playwright-core/src/utils/network.ts

```diff
@@ -82,7 +82,7 @@ export function httpRequest(params: HTTPRequestParams, onResponse: (r: http.Inco
   request.on('error', onError);
   if (timeout !== undefined) {
     const rejectOnTimeout = () =>  {
-      onError(new Error(`Request to ${params.url} timed out after ${timeout}ms`));
+      onError(new Error(`Request to ${params.url} timed out after ${timeout}ms. Consider setting PLAYWRIGHT_DOWNLOAD_CONNECTION_TIMEOUT if you haven't already.`));
```

Comment:
Is this a really necessary message? In my opinion, information like that is more suitable in a documentation of FAQ